.travis.yml

```diff
@@ -1,10 +1,12 @@
 language: python
 
 python:
-    - "2.6"
     - "2.7"
-    - "3.3"
     - "3.4"
+    - "3.5"
```

Comment:
Build failed for the older versions as not available on travis. I've added the newer versions as the build is able to pass for the new ones.